deps/download.sh

```diff
@@ -77,18 +78,18 @@ tar czf "$DEPS/sqlite3.tar.gz" sqlite3.c sqlite3.h sqlite3ext.h || exit 1
 
 echo "updating gyp configs..."
 GYP="$DEPS/defines.gypi"
-printf "# THIS FILE IS AUTOMATICALLY GENERATED (DO NOT EDIT)\n\n{\n  'defines': [\n" > "$GYP"
-printf "$DEFINES" | sed -e "/^\s*$/d" -e "s/\(.*\)/    '\1',/" >> "$GYP"
+printf "# THIS FILE IS AUTOMATICALLY GENERATED (DO NOT EDIT)\n\n{\n  'defines': [\n" >"$GYP"
+printf '%s' "$DEFINES" | sed -e "/^\s*$/d" -e "s/\(.*\)/    '\1',/" >>"$GYP"
 printf "  ],\n}\n" >> "$GYP"
```

Comment:
Why did shellcheck notice some of these but not other cases of ">>" with a trailing space character?

Reply:
Dang, I thought I'd reverted all of the whitespace deltas: they're from `prettier`.